adding_service_cluster/rosa-available-services.adoc

```diff
@@ -15,7 +15,26 @@ include::modules/aws-cloudwatch.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 
+* link:https://aws.amazon.com/cloudwatch/[Amazon CloudWatch product information]
 * xref:../rosa_cluster_admin/rosa_logging/rosa-install-logging.adoc#rosa-install-logging[Installing the CloudWatch logging service]
-* link:https://aws.amazon.com/cloudwatch/[Amazon CloudWatch product information] 
+* link:https://aws.amazon.com/cloudwatch/[Amazon CloudWatch product information]
```

Comment:
You have this link listed twice here

Reply:
I think that's a code hiccup, I was removing a non-breaking space at the end of the link. In the preview, there's one link. :+1: